app/views/efficacy/EfficacyView.vue

```diff
@@ -0,0 +1,974 @@
+          p.small Student, McIntosh Middle School
+          h1 &rdquo;
+
+    section.container.teacher-quote
+      .bubble
+        p I have absolutely loved using Ozaria in my class for the past three years. My students have thoroughly enjoyed the storyline as they learned Python in a self-paced learning environment. I would highly recommend Ozaria for teachers that know little to nothing about coding, as the support for teachers is very in-depth.
+        p.small
+          | Lynette Personett, Teacher
+          br
+          | Mountain Ridge Middle School
+        .image
+          img(src="/images/pages/efficacy/teacher-2.png")
+
+    section.full-width.section-commitment
+      .container
+        h2 CodeCombat’s Commitment to Efficacy
+        p We are committed to ensuring our products have a positive and measurable impact on learning outcomes. We create meaningful learning experiences, and use evidence-based practices to design content that will realize the outcomes that matter the most to teachers and students.
+        p.question Interested in implementing an effective and impactful CS curriculum at your school?
+        a(@click="onClickContact") Contact Us
+
+    section.teachers-love
+      .container
+        h2 Teachers Love CodeCombat
+        p Explore the impact of our sister product, CodeCombat, on Computer Science teaching & learning!
+        .row.teachers-love-boxes
+          .col.col-md-4.teachers-love-box
+            h3 99%
+            p Report that their students enjoy using CodeCombat to learn how to code
+          .col.col-md-4.teachers-love-box
+            h3 95%
+            p Would recommend CodeCombat to other computer science teachers
+          .col.col-md-4.teachers-love-box
+            h3 94%
+            p Say that CodeCombat helps them support students’ problem solving abilities
+        a(href="https://codecombat.com/impact") Learn More
+
+    modal-get-licenses(v-if="showModalGetLicenses" @close="showModalGetLicenses = false" subtitle="To get licenses for our professional development course, send us a message and our classroom success team will be in touch!" email-message="Hi Ozaria! I'm interested in learning more about Ozaria's research-backed computer science curricula.")
+</template>
+
+<style lang="scss" scoped>
+@import "app/styles/bootstrap/variables";
+@import "app/styles/mixins";
+@import "app/styles/style-flat-variables";
+@import "ozaria/site/styles/common/common.scss";
+@import "app/styles/ozaria/_ozaria-style-params.scss";
+
+.dialog {
+  position: absolute;
+  top: 198px;
+  left: 986px;
+  background: url(/images/pages/efficacy/code-box.svg);
+  width: 247px;
+  height: 132px;
+  padding: 18px 22px;
+  span, p {
+    font-family: 'Space Mono';
```

Comment:
Let's add a font stack here (ideally in a variable) so that it will still work even if we haven't loaded Space Mono. Similar comment to https://github.com/codecombat/codecombat/pull/6605#discussion_r880833410 which was adjusted in https://github.com/codecombat/codecombat/pull/6605/commits/8ce66ea757c191d40426dc1d30a60cc3021f102c#diff-045a09881f4c41285ac1b13c26dc45e489e826d14c040289717ac19bf712337cR5